data/templates/esx-ks

```diff
@@ -236,7 +214,9 @@ cp /var/log/esxi_install.log "/vmfs/volumes/datastore1/firstboot-esxi_install.lo
 #
 # NOTE: this script will execute right away as a result of writing it to local.sh
 # along with executing on every subsequent boot
-wget http://<%=server%>:<%=port%>/api/current/templates/<%=rackhdCallbackScript%> -O /etc/rc.local.d/local.sh
+# rackhdCallbackScript is no longer in context once "Task.Os.Install.ESXi" is marked as completed
+
```

Comment:
Yes but the wget itself will fail because the esx.rackhdcallback file containsvariables to be rendered as well.